browser/views.py

```diff
@@ -688,6 +689,44 @@ def delete_investigator_record(request, person_id, research_id):
 
 
 @staff_member_required
+def add_person(request):
+    template = "browser/add_person.html"
+    form = PersonForm()
+    knownPersonForm = KnownPersonForm()
+    context = {'form': form,
+               'knownPersonForm': knownPersonForm}
+    if request.method == 'POST':
+        form = PersonForm(request.POST)
+        knownPersonForm = KnownPersonForm(request.POST)
+        context.update(
+            {'form': form}
+        )
+        if form.is_valid():
+            # Check if the person already exists in the database
+            if Person.objects.filter(first_name=form.cleaned_data.get('first_name'),
+                    last_name=form.cleaned_data.get('last_name')).exists():
```

Comment:
just because two people have the same name, doesn't mean they are the same person

Reply:
Removed this check